src/frontend/pages/Testnet.tsx

```diff
@@ -1 +1,8 @@
-export const Testnet = () => <div>Testnet</div>;
+import { NetworkStatus } from "frontend/components/NetworkStatus";
+import { Network } from "types";
+
+export const Testnet = () => (
+  <div className="ContentWrapper">
+    <NetworkStatus network={Network.TESTNET} />
```

Comment:
I thought it will be more efficient to pass the testnet prop manually to components instead of detecting it dynamically on every component (default network is mainnet, so no need to pass it there). We already know the network where the components are used.